content/en/serverless/guide/serverless_tracing_and_bundlers.md

```diff
@@ -126,6 +126,78 @@ Datadog's tracing libraries (`dd-trace`) are known to be not compatible with bun
     module.exports = [nodeExternalsPlugin()]
     ```
 
+## AWS CDK & esbuild
```

Comment:
Would it make sense to cover customers who are only using the CDK configuration without esbuild?

Reply:
Yeah, that's not a bad shout. I'll update that today.

Reply:
@duncanista I've added an additional section. Let me know if that looks right, I've tested the code and I still get all the tracing.